cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/SpaceDevelopersRequestsPlaner.java

```diff
@@ -18,56 +14,35 @@
 
 /**
  * This class is responsible to build the requests in the context of space developers according to the CfChanges.
- * The class creates the request tasks by implementing the {@link CfChangeVisitor} interface.
  */
-public class SpaceDevelopersRequestsPlaner implements CfChangeVisitor {
+public class SpaceDevelopersRequestsPlaner {
 
     private static final Log log = Log.getLog(SpaceDevelopersRequestsPlaner.class);
 
     private final SpaceDevelopersOperations spaceDevelopersOperations;
     private final List<Mono<Void>> requests;
 
-    private SpaceDevelopersRequestsPlaner(SpaceDevelopersOperations spaceDevelopersOperations) {
+    public SpaceDevelopersRequestsPlaner(SpaceDevelopersOperations spaceDevelopersOperations) {
         this.spaceDevelopersOperations = spaceDevelopersOperations;
         this.requests = new LinkedList<>();
     }
```

Comment:
You could even remove the constructor and offer a single static method for creating the space developer requests instead.